http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcJsonConfiguration.java

```diff
@@ -41,12 +41,13 @@
         "expose-token", "bearer-only", "autodetect-bearer-only",
         "connection-pool-size",
         "allow-any-hostname", "disable-trust-manager", "truststore", "truststore-password",
-        "client-keystore", "client-keystore-password", "client-key-password",
+        "client-keystore-file", "client-keystore-password", "client-key-password", "client-key-alias", "client-keystore-type",
```

Comment:
@PrarthonaPaul Just a minor, should the "client-keystore" stay here as possible configuration property because of backwards compatibility? Users could have been using it before?